.swiftlint.yml

```diff
@@ -107,4 +107,4 @@ line_length:
   warning: 500
   error: 500
 
-swiftlint_version: 0.54.0
+swiftlint_version: 0.55.1
```

Comment:
This doubles the work @agisilaos  and me did already on some branch recently (I lost track which one). But thanks you updated everything also according to the new rules, we just disabled it 😁